docs/CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 Antares Changelog
 =================
 
+v8.6.0-rc6 (05/2023)
+--------------------
+## Format improvement
+ * Add bool for initial level optimization (#1312)
```

Comment:
which levels are we talking about ?

Reply:
#1312 as requested by @laurent-laporte-pro

Reply:
done